Cargo.toml

```diff
@@ -44,3 +44,5 @@ harness = false
 codegen-units = 1
 lto = true
 
+[profile.test]
+debug-assertions = true
```

Comment:
I am curious why this change was done. Test is by default dev and that's with debug assertions? https://doc.rust-lang.org/cargo/reference/profiles.html#test

Reply:
Oh, I forgot to remove this. I was debugging why the assertion didn't trigger when I tested locally. It turned out that the unicode-width in my local lock file was still the old version.